server.py

```diff
@@ -186,6 +208,10 @@ def handle_disconnect_event():
 # Triggered when the client sends a command
 @socketio.on('arduino_command')
 def handle_arduino_command(command):
+    if(request.remote_addr not in serverconfig.authorized_ips):
```

Comment:
We definitely could, I was actually going to ask you about that. I guess one justification to leave it this way is if we want to differentiate between who can view the data and who can also execute commands. Just let me know if you prefer to have it changed.